### ISSUE TITLE
Support for low power co proccesor in 3100

### ISSUE THREAD

**Issue by NicholasBM** [CorvetteCole/PixelWatchFace#14]
Hey there 

It would be amazing if you were able to take advantage of the co proccesor for your ambient mode. I would hope that would improve battery life on all of the newer devices. 

There was a blog about it here 

http://turndapage.com/2018/12/07/implementing-the-new-ambient-second-hand-for-wear-os-with-watch-face-decompositions/

Thanks 

Nick

**Comment by CorvetteCole** [comment]
good idea, I will add a second indicator for newer devices in a future update
